buildconfig/pygame-stubs/joystick.pyi

```diff
@@ -5,7 +5,7 @@ def quit() -> None: ...
 def get_init() -> bool: ...
 def get_count() -> int: ...
 
-class Joystick(object):
```

Comment:
Is (object) needed for py2 support?

Reply:
Yes, will put all those back. 

Are there many developers using Python 2 for Pygame? I noticed there were some tutorials that used Python 2. But was not clear about the prevlaence.